cpp/src/Slice/Parser.cpp

```diff
@@ -853,6 +848,7 @@ Slice::Contained::parseComment(const string& text, bool stripMarkup) const
         {
             if (!line.empty())
             {
+                state = StateMisc; // Reset the state; `@see` cannot span multiple lines.
```

Comment:
Fixes a bug where we'd sometimes generate multi-line `see` statements.
These should always be a single line like: `@see MyType.op79`